assets/src/block-templates/reality-check/template.js

```diff
@@ -0,0 +1,32 @@
+import metadata from './block.json';
+import mainThemeUrl from '../main-theme-url';
+
+const column = ['core/column', {}, [
+  ['core/group', {}, [
+    ['core/image', {
+      align: 'center',
+      className: 'mb-0 force-no-lightbox force-no-caption',
+      url: `${mainThemeUrl}/images/placeholders/placeholder-75x75.jpg`
+    }],
+    ['core/heading', {
+      style: {typography: {fontSize: '4rem'}},
+      textAlign: 'center',
+      placeholder: __('Enter title', 'planet4-blocks-backend')
+    }],
+    ['core/paragraph', {
+      align: 'center',
+      placeholder: __('Enter description', 'planet4-blocks-backend')
+    }],
+    ['core/spacer', {height: '16px'}],
+  ]]
+]];
+
+const template = () => ([
+  ['core/columns', {
+    className: `block ${metadata.classname}`
```

Comment:
This `metadata.classname` results in `undefined`, and it seems that we don't need it so I think we shouldn't add it 🤔 I had added it to my Quick Links conversion too, but I'm removing it in [this PR](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/971)

Reply:
Thanks, done!!

Reply:
Thanks, done!!